deployment_cfn/_layout_deployment.adoc

```diff
@@ -45,7 +45,7 @@ endif::production_build[]
 
 == Deployment options
```

Comment:
```suggestion
== Choose a deployment option
```
This updated heading reflects the new actionable nature of this section's content.

Reply:
Keeping as is to avoid confusion about when to start a deployment. Seasoned devs will know which parts they can skip.